docs/docs/deploy/environment-variables.mdx

```diff
@@ -0,0 +1,131 @@
+| `TICKET_LOCK_LIFETIME`            | The lifetime of ticket locks, in seconds. It configures the [lock store](../production/lock-stores.mdx).                                                                                                              | `60`              |
+| `VAULT_URL`                       | <RasaProLabel /> The URl of the [HashiCorp Vault](../production/secrets-managers.mdx#hashicorp-vault-secrets-manager).                                                                                                |                   |
+| `VAULT_TOKEN`                     | <RasaProLabel /> The token to authenticate to the [HashiCorp Vault](../production/secrets-managers.mdx#hashicorp-vault-secrets-manager).                                                                              |                   |
+| `VAULT_NAMESPACE`                 | <RasaProLabel /> The namespace of the [HashiCorp Vault](../production/secrets-managers.mdx#hashicorp-vault-secrets-manager).                                                                                          |                   |
+| `VAULT_RASA_SECRETS_PATH`         | <RasaProLabel /> The path to Rasa secrets in the [HashiCorp Vault](../production/secrets-managers.mdx#hashicorp-vault-secrets-manager).                                                                               | `"rasa-secrets"`  |
+| `VAULT_TRANSIT_MOUNT_POINT`       | <RasaProLabel /> The mount point of the [HashiCorp Vault](../production/secrets-managers.mdx#hashicorp-vault-secrets-manager).                                                                                        |                   |
+
+### Model Storage
+
+Environment variables that configure model storage on cloud providers are described in the following sections:
+- [Amazon S3 Storage](../production/model-storage.mdx#amazon-s3-storage)
+- [Google Cloud Storage](../production/model-storage.mdx#google-cloud-storage)
+- [Azure Storage](../production/model-storage.mdx#azure-storage)
+
+### Dialogue Management
+
+| Environment Variable             | Description                                                                                                                                                                                   | Default   |
+| :------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :-------- |
+| `RASA_DUCKLING_HTTP_URL`          | The URL of the Duckling service powering the [`DucklingEntityExtractor`](../nlu-based-assistants/components.mdx#ducklingentityextractor).                                                    |           |
```

Comment:
I remember seeing an envar for action server URL too but it isn't mentioned on the notion page and I don't see it in the code either. Do you remember anything like that?

Reply:
52e91961904

Reply:
That's a new one yep, we're good here I think